lib/puppet/provider/keycloak_client/kcadm.rb

```diff
@@ -222,7 +229,12 @@ def flush
           unless data.key?(:attributes)
             data[:attributes] = {}
           end
-          data[:attributes][property] = value
+          if dot_attributes_properties.include?(property)
```

Comment:
Something like this will also be needed in the `create` method.

Reply:
Of course. I'm on it.